docs/MVS_Outputs.rst

```diff
@@ -155,6 +155,47 @@ To measure this, we propose to compare the energy flows in between the sectors t
 
 ** The content of this section was copied from the conference paper handed in to CIRED 2020**
 
+Self consumption (SC)
+###############################
+
+The self consumption describes the fraction of all renewable energy that is consumed by the system itself.
+
+.. math::
+        SC &=\frac{\sum_{i,j} {E_{generation} (i,j) \cdot w_i} - E_{gridfeedin}(i,j) \cdot w_i}{\sum_{i,j} {E_{generation} (i,j) \cdot w_i}}
+
+        \text{with~} &i \epsilon \text{[PV,Geothermal,…]}
+
+        &j \epsilon \text{[generation assets 1,2,…]}
+
+[1] https://www.sciencedirect.com/science/article/pii/S0960148119315216
+[2] https://www.iip.kit.edu/downloads/McKennaetal_paper_full.pdf
+
+self_sufficiency (SS)
+###############################
```

Comment:
```suggestion
Self-sufficiency (SS)
###############
```
The line of "#" needs to be as long as your title, otherwise there are error messages. 

Is SS a commonly used abbreviation?